scripts/warn-outside-container

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+set -eu
+
+target="${1:-}"
+
+if [[ -z "${DISABLE_WARN_OUTSIDE_CONTAINER:-}" ]]; then
+    (
+        echo
+        echo
+        echo "WARNING: you are not in a container."
+        echo "Use \"make -f docker.Makefile $target\" or set"
+        echo "DISABLE_WARN_OUTSIDE_CONTAINER=1 to disable this warning."
+        echo
+        echo
+    ) >&2
+    sleep 5
```

Comment:
> Should we just exit here?

`exit` won't actually stop the make target from running. There's no way (that I could find) to force a target before every other target in a Makefile. We'd have to add it manually as a dependency to everything, and I don't think that's a good solution.

So this is implemented as just a shell call `$(shell ...)` which means we get stdout/stderr, but can't actually end the execution. Maybe there's some way to have to exit, but I couldn't find a way to do it with make.